backend/tests/e2e/test_api_e2e.py

```
@@ -0,0 +1,860 @@
+"""
+Focused End-to-End Integration Test for HTTP API Endpoints.
+
+This test validates the complete alert processing pipeline with real LLM/MCP interactions,
+database persistence, and comprehensive API data structures.
+
+Uses isolated e2e test fixtures to prevent interference with unit/integration tests.
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)